src/io-methods/azure.js

```diff
@@ -1,38 +1,52 @@
-const azure = require("azure-storage");
+const { BlobServiceClient } = require("@azure/storage-blob");
 let blobService;
 
-function getBlobService() {
-  blobService = blobService || azure.createBlobService();
+const AZURE_STORAGE_CONNECTION_STRING =
+  process.env.AZURE_STORAGE_CONNECTION_STRING;
+
+async function getBlobService() {
+  blobService =
+    blobService ||
+    (await BlobServiceClient.fromConnectionString(
+      AZURE_STORAGE_CONNECTION_STRING
```

Comment:
This appears to be a breaking change - the documentation at https://github.com/single-spa/import-map-deployer#azure-storage indicates that `AZURE_STORAGE_ACCOUNT` and `AZURE_STORAGE_ACCESS_KEY` environment variables can also be used as an alternative to `AZURE_STORAGE_CONNECTION_STRING`. However, your change requires AZURE_STORAGE_CONNECTION_STRING to be present.

Can we make this work with either set of env variables? Am looking at https://docs.microsoft.com/en-us/javascript/api/%40azure/storage-blob/blobserviceclient?view=azure-node-latest#createcontainer-string--containercreateoptions-